src/time.rs

```diff
@@ -60,3 +55,32 @@ impl Time {
     /// used).
     pub fn from_seconds_since_unix_epoch(secs: u64) -> Time { Time(secs) }
 }
+
+#[cfg(feature = "std")]
+impl std::convert::TryFrom<::std::time::SystemTime> for Time {
```

Comment:
Please use `core::convert::TryFrom`. Although it doesn't matter much, it makes things clearer when we search for uses of libstd in the codebase, which we'll be doing frequently in the near future because of the desire to add support for some no_std targets.

Reply:
Sounds good, don't want to be distracted by superfluous `std` uses 🙂